.gitignore

```diff
@@ -1,2 +1,3 @@
 .DS_Store
 .idea/
+.vscode
```

Comment:
Not a big fan of having editor/IDE configs in `.gitignore`. I prefer global setting (https://help.github.com/articles/ignoring-files/#create-a-global-gitignore). But due to the fact we already have `.idea` I can not complain :-) ... Just keep in mind that eventually those lines might be removed from `.gitignore`.